src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardSamlBundle/Security/Authentication/Provider/SamlProvider.php

```diff
@@ -48,19 +49,28 @@ class SamlProvider implements SamlProviderInterface, UserProviderInterface
      */
     private readonly array $administratorTeams;
 
+    private readonly string $surfConextResponsibleAuthorization;
+
     public function __construct(
         private readonly ContactRepository $contacts,
         private readonly ServiceRepository $services,
         private readonly AttributeDictionary $attributeDictionary,
-        private readonly LoggerInterface $logger,
+        private readonly LoggerInterface  $logger,
```

Comment:
Why the extra space?

Reply:
Reverted that. Note that it was not a change made by me. But somehow got triggered on this PR :man_shrugging:  Probably because I moved the params around a bit..